tests/network/vmi_networking.go

```diff
@@ -204,30 +203,26 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 
 				ip := inboundVMI.Status.Interfaces[0].IP
 
-				tcpJob := job.NewHelloWorldJobTCP(ip, strconv.Itoa(testPort))
-				tcpJob.Spec.Template.Spec.Affinity = &k8sv1.Affinity{
-					NodeAffinity: &k8sv1.NodeAffinity{
-						RequiredDuringSchedulingIgnoredDuringExecution: &k8sv1.NodeSelector{
-							NodeSelectorTerms: []k8sv1.NodeSelectorTerm{
-								{
-									MatchExpressions: []k8sv1.NodeSelectorRequirement{
-										{Key: k8sv1.LabelHostname, Operator: op, Values: []string{inboundVMI.Status.NodeName}},
-									},
-								},
-							},
-						},
-					},
-				}
-				tcpJob.Spec.Template.Spec.HostNetwork = hostNetwork
+				By("start connectivity job on the same node as the VM")
+				localNodeTCPJob := job.NewHelloWorldJobTCP(ip, strconv.Itoa(testPort))
+				localNodeTCPJob.Spec.Template.Spec.HostNetwork = hostNetwork
+				localNodeTCPJob.Spec.Template.Spec.Affinity = &k8sv1.Affinity{NodeAffinity: newNodeAffinity(inboundVMI.Status.NodeName)}
+				localNodeTCPJob, err = virtClient.BatchV1().Jobs(inboundVMI.ObjectMeta.Namespace).Create(context.Background(), localNodeTCPJob, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
 
-				tcpJob, err = virtClient.BatchV1().Jobs(inboundVMI.ObjectMeta.Namespace).Create(context.Background(), tcpJob, metav1.CreateOptions{})
+				By("start connectivity job on different node")
+				remoteNodeTCPJob := job.NewHelloWorldJobTCP(ip, strconv.Itoa(testPort))
+				remoteNodeTCPJob.Spec.Template.Spec.HostNetwork = hostNetwork
+				remoteNodeTCPJob.Spec.Template.Spec.Affinity = &k8sv1.Affinity{PodAntiAffinity: newPodAntiAffinity(testLabel)}
+				remoteNodeTCPJob, err = virtClient.BatchV1().Jobs(inboundVMI.ObjectMeta.Namespace).Create(context.Background(), remoteNodeTCPJob, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				Expect(job.WaitForJobToSucceed(tcpJob, 90*time.Second)).To(Succeed())
+
+				Expect(job.WaitForJobToSucceed(remoteNodeTCPJob, 90*time.Second)).To(Succeed(), "should be able to reach VM workload from a pod on different node")
+				Expect(job.WaitForJobToSucceed(localNodeTCPJob, 90*time.Second)).To(Succeed(), "should be able to reach VM workload from a pod on the same node")
+
 			},
-				Entry("[test_id:1543]on the same node from Pod", k8sv1.NodeSelectorOpIn, false),
-				Entry("[test_id:1544]on a different node from Pod", k8sv1.NodeSelectorOpNotIn, false),
-				Entry("[test_id:1545]on the same node from Node", k8sv1.NodeSelectorOpIn, true),
-				Entry("[test_id:1546]on a different node from Node", k8sv1.NodeSelectorOpNotIn, true),
+				Entry("[test_id:1544] from pod network", false),
+				Entry("[test_id:1543] from node network", true),
```

Comment:
What about dropping the tests of the node-network?
Why do we need it?

Reply:
This PR scope covers tests over the pod network and doesnt account for the host-network cases.

The host network tests seem pretty old and to be honest I dont know why we need them.
Kubevirt perfomes the plumbing inside the pod, while testing host-network is the CNI area.

If you think we can drop them I dont mind doing thins as part of this PR.

Reply:
Please drop them.

Reply:
Done